src/http.js

```diff
@@ -25,6 +25,7 @@ export default function http(url, request) {
   if (request.requestInterceptor) {
     request = request.requestInterceptor(request) || request
   }
+  request.credentials = 'same-origin';
```

Comment:
Setting this to `'include'` works when it's cross-origin and when CORS is allowed.